inst/adducts/adduct_definition.txt

```diff
@@ -48,3 +48,7 @@
 "[2M+CHO2]-"	2	44.9982	"CHO2"	"C0"	-1	FALSE
 "[2M+C2H3O2]-"	2	59.01385	"C2H3O2"	"C0"	-1	FALSE
 "[3M-H]-"	3	-1.007276	"C0"	"H"	-1	FALSE
+"[M+H-NH3]+"	1	-16.01927265	"H"	"NH3"	1	TRUE
+"[M-H+HCOOH]-"	1	44.998194	"HCOOH"	"H"	-1	FALSE
```

Comment:
This is the same as [M+CHO2]- above. This is a huge problem of non-standardized adduct names

Reply:
Great! Thanks @michaelwitting! That's why I wanted to let you have a look at this. 
So I drop the M-H+HCOOH but keep the M+H-NH3?